File: main/src/main/java/net/fwitz/math/main/binary/complex/functions/examples/simple/IdentityMain.java
package net.fwitz.math.main.binary.complex.functions.examples.simple;

import net.fwitz.math.plot.binary.complex.ComplexFunctionPlot;
import net.fwitz.math.plot.binary.complex.color.DomainColoringContour;

public class IdentityMain {
    public static void main(String[] args) {
        new ComplexFunctionPlot("z")
                .computeFn(z -> z)
                .colorFn(new DomainColoringContour())
                .render();
    }
}

File: main/src/main/kotlin/net/fwitz/math/main/binary/complex/functions/examples/simple/IdentityMain.kt
package net.fwitz.math.main.binary.complex.functions.examples.simple

import net.fwitz.math.plot.binary.complex.ComplexFunctionPlot
import net.fwitz.math.plot.binary.complex.color.DomainColoringContour

object IdentityMain {
    @JvmStatic
    fun main(args: Array<String>) = ComplexFunctionPlot("z")
        .computeFn { z -> z }
        .colorFn(DomainColoringContour)
        .render()
}